.github/workflows/build.yml

```diff
@@ -132,6 +133,12 @@ jobs:
           git -C ../${{ env.NEOS_FOLDER }} checkout -b build
           composer config repositories.neos '{ "type": "path", "url": "../${{ env.NEOS_FOLDER }}", "options": { "symlink": false } }'
           composer require --no-update neos/neos-development-collection:"dev-build as ${{ env.NEOS_BRANCH_ALIAS }}"
+          # TODO REMOVE BEFORE MERGE!!!!!
```

Comment:
Mmm, I guess this neesd to happen still?

Reply:
Yes always these details and hacks used to fix ci for those complex things 🙈 ...